CHANGELOG.md

```diff
@@ -31,9 +31,18 @@ Thank you to all who have contributed!
 - Adds top-level IR node creation functions.
 - Adds `componentN` functions (destructuring) to IR nodes via Kotlin data classes
 - Adds public `tag` field to IR nodes for associating metadata
+- Adds AST Normalization Pass. 
+- Adds PartiQLPlanner Interface, which is responsible for translate an AST to a Plan. 
 
 ### Changed
 - StaticTypeInferencer and PlanTyper will not raise an error when an expression is inferred to `NULL` or `unionOf(NULL, MISSING)`. In these cases the StaticTypeInferencer and PlanTyper will still raise the Problem Code `ExpressionAlwaysReturnsNullOrMissing` but the severity of the problem has been changed to warning. In the case an expression always returns `MISSING`, problem code `ExpressionAlwaysReturnsMissing` will be raised, which will have problem severity of error.
+- The default integer literal type is now 32-bit. 
+- `PartiQLValueType` now distinguishes between Arbitrary Precision Decimal and Fixed Precision Decimal. 
+- Function Signature Changes. Now Function signature has two subclasses, `Scalar` and `Aggregation`. 
+- In the produced plan: 
+  - The new plan is fully resolved and typed.
+  - Operators will be converted to function call. 
```

Comment:
Can you label all of these as breaking? We really need to call out the literal integer behavior change and the public API changes.

Reply:
Integer typing uses 32-bit first, overflows to 64-bit if too big, then finally arbitrary size backed by Java BigInt

Reply:
> finally arbitrary size backed by Java BigInt 
I don't recall we have this implemented.

Reply:
https://github.com/partiql/partiql-lang-kotlin/blob/partiql-planner/partiql-parser/src/main/kotlin/org/partiql/parser/impl/PartiQLParserDefault.kt#L1934-L1960